_posts/2022-02-01-Midi-Mixerboard-Control.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Control the Jamulus Mixerboard using MIDI"
+lang: "en"
+author: "henkdegroot"
+heading: "How to control the Jamulus Mixerboard using a MIDI Control Surface in Windows using JACK"
+---
+
+This guide explains how to setup your system in order to control the Jamulus Mixerboard using a MIDI Control Surface device (or similar) using JACK.
+The process will be similar for Linux (this also requires JACK and a2jmidid), while for macOS you can use MIDI Studio.
+
+## What do you need?
+1. Jamulus with JACK support (not the standard ASIO version), you can [download the Jamulus JACK version for Windows here]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }})).
+2. You need an audio device with ASIO support or you can use a universal ASIO driver like ASIO4ALL.
+3. You need a MIDI Control Surface. Many different models are available. I am using a Worlde EasyControl 9.
+ ![worlde-easycontrol-9](https://user-images.githubusercontent.com/13550012/152056661-a446f24e-0598-4164-bc0f-a2510ab33c5e.png)
+4. JACK for Winodws [get it from the JACK audio page](https://jackaudio.org/downloads/).
```

Comment:
Usually not reviewing KB posts, but there’s a typo (it should be Windows).

Also maybe focus a bit less on Windows. State you‘re using Windows as example, but refer to other OS

Reply:
Appreciate you took the time to review this!